modules/_node-scripts/configuration/getProjectDescription.mjs

```diff
@@ -0,0 +1,14 @@
+import require from '../util/require.mjs';
+
+/**
+ * @returns 
+ * {
+ *	 name: 'xxx',
+ *	 version: 'x.y.z',
+ * }
+ */
+export default function getProjectDescription() {
+	const {main, name, version} = require('./package.json');
+
+	return {main, name, version};
+}
```

Comment:
Why have a helper like this? Isn't this the exact same as if we just imported the package.json itself?

Reply:
Well, simply to be able to load it just once in the configuration phase. These values are only needed for legacy (AMD) stuff and if we need to access the project name in the future we may even obtain it from a different file (eg: the `bnd.bnd` file) since there's no need to have `package.json` files in the ESM model. 

So I think it makes sense to have a kind of abstract function to get the project description...